blog/2024-08-15-christian-fritz-rostool/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "Christian Fritz presents ROS Tool"
+slug: christian-fritz-rostool
+authors: michaelhart
+tags: [ros, rostool, guest]
+---
+
+[Christian Fritz](https://www.linkedin.com/in/christianfritz), Founder and CEO at [Transitive Robotics](https://transitiverobotics.com/), is a valued member of the Cloud Robotics Working Group. During our [meeting on 2024-07-29](/meetings), Christian presented to the group on [ROS Tool](https://transitiverobotics.com/caps/transitive-robotics/ros-tool/), a tool built by Transitive Robotics that adds a new way to communicate between robot cloud and web specifically for ROS. It is now a capability of Transitive Robotics, but has been made [open source](https://github.com/transitiverobotics/transitive) for anyone to use.
```

Comment:
Suggested edit:

During our [meeting on 2024-07-29](/meetings), [Christian Fritz](https://www.linkedin.com/in/christianfritz), Founder and CEO at [Transitive Robotics](https://transitiverobotics.com/) and a member of the Cloud Robotics Working Group, presented [ROS Tool](https://transitiverobotics.com/caps/transitive-robotics/ros-tool/), a tool built by Transitive Robotics that adds a new way to communicate between robot, cloud, and web specifically for ROS. It is now a capability of Transitive Robotics you can try out. It also show-cases some of the design decisions and benefits of the [open-source Transitive framework](https://github.com/transitiverobotics/transitive).